framework/include/base/MooseApp.h

```diff
@@ -1582,7 +1585,7 @@ class MooseApp : public ConsoleStreamInterface,
   std::unique_ptr<Backup> * const _initial_backup;
 
 #ifdef MOOSE_LIBTORCH_ENABLED
-  /// The libtorch device this app is using.
+  /// The libtorch device this app is using (converted from _compute_device)
```

Comment:
```suggestion
  /// The libtorch device this app is using (converted from compute_device)
```

Reply:
both are correct, but only `_compute_device` exists in this header file, so I felt it to be more appropriate to reference that

Reply:
oh, wait, I got rid of `_compute_device`, didn't I?